app/models/payment.rb

```diff
@@ -0,0 +1,12 @@
+class Payment < ApplicationRecord
+  belongs_to :user
+  belongs_to :course
+
+  PAYMENT_TYPES = { mobile: 0, bank: 1 }.freeze
+  enum payment_type: PAYMENT_TYPES
+
+  STATUS_TYPES = { unpaid: 0, paid: 1 }.freeze
+  enum status: STATUS_TYPES
```

Comment:
ditto

Reply:
Apu, actually Sarah apu suggested this: https://github.com/amu-wtag/well_academy/pull/2#discussion_r1839875861 . Should I keep it or change?